src/firefly/js/visualize/CsysConverter.js

```diff
@@ -630,7 +630,7 @@ export class CysConverter {
     makeWorldPtFromIPt( ipt, outputCoordSys) {
         if (!ipt) return null;
         let wpt = this.projection.getWorldCoords(ipt.x - .5 ,ipt.y - .5);
-        if (wpt && outputCoordSys!==wpt.getCoordSys()) {
+        if (wpt && outputCoordSys && outputCoordSys!==wpt.getCoordSys()) {
```

Comment:
Agree with you, I changed this before I found out the true issue.  I should have reviewed it and changed it back. This is not an issue at all since the covert has a default argument.